test/integration/conftest.py

```diff
@@ -140,7 +132,7 @@ def container_image(request, cli, tmp_path):  # pylint: disable=W0621
     image_name = f'ansible-runner-{random_string}-event-test'
 
     cli(
-        [runtime, 'build', '--build-arg', f'WHEEL={wheel.name}', '--rm=true', '-t', image_name, '-f', str(dockerfile_path), str(tmp_path)],
+        [runtime, 'build', '--build-arg', '--rm=true', '-t', image_name, '-f', str(dockerfile_path), str(tmp_path)],
```

Comment:
Is this breaking things?
```suggestion
        [runtime, 'build', '--rm=true', '-t', image_name, '-f', str(dockerfile_path), str(tmp_path)],
```

Reply:
Good spot. No, didn't break anything, simply ignored.